packages/core/core/src/Environment.js

```diff
@@ -13,10 +13,12 @@ export function createEnvironment({
   engines,
   includeNodeModules,
   outputFormat,
+  sourceType,
```

Comment:
(assuming `null` is a desirable type for `SourceMap`)

```suggestion
  sourceType = 'module',
```